system/mp/mpcomm.cpp

```diff
@@ -756,8 +760,7 @@ protected: friend class CMPPacketReader;
                 // loop with short wait time and release CS to allow other side to proceed
                 StringBuffer epStr;
                 unsigned startMs = msTick();
-
-                unsigned loopCnt = ((CONNECT_READ_TIMEOUT / retrycount) / CONNECT_TIMEOUT_INTERVAL) + 1;
+                unsigned loopCnt = CONNECT_TIMEOUT_LOOPCNT;
```

Comment:
The intervals are still shorter..

I assume that the rising time in each retry pass was unintentional (I think you confirmed)
I also assume that the overall descrese in time from the original code, which was 10 * 180 secs was unintentional.

Using a fixed loopCnt of 180 (= 180secs) restores the original overall time.. *except* the retriesCount was upped to 20 instead of 10 , and if there was no other reason , it should also be restored afaics.